### PR TITLE
Add separate worker for logging & excluded THUB_TOKEN_IS_VALID from env

### LINKED ISSUE #784: Implement logging of Terrahub workflow  to local file
# Feature Report
Add functionality to have workflow logs in all use cases (ES or local Log files)
## Describe the Feature

Implement logging to local file ``` ~/.terrahub/cache/log/[runId]/[componentName].log ``` if ES logging is disabled 

## Expected Behavior
 

Logging flow : 
on start terrahub cli - create [runId]_start.log
on start terrahub component - create [runId]_component.log, upload [runId]_start.log
on end terrahub component - create [runId]_end.log, upload [runId]_component.log
on end terrahub cli  - merge all logs into 1 file, upload [runId]_end.log

### PR DESCRIPTION
**Pull request by const-andreyLu**
## Description

Fixes #784, #794

## Type of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a change to the documentation

## How Has This Been Tested?

- [x] terrahub run (with/without token, with/without aws.creds)
- [x] terrahub destroy (with/without token, with/without aws.creds)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have checked that my changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/helpers/fetch.js [hunk 1]
```diff
@@ -73,6 +73,14 @@ class Fetch {
   _handleResponse(result) {
     return result.json().then(json => {
 
+
+      fs.appendFileSync('/Users/andreyluchianic/Documents/log.txt', JSON.stringify({
```
Comment (eistrati) [inline src/helpers/fetch.js]: please replace `/Users/andreyluchianic/Documents/log.txt` with something that is more user friendly

Comment (euliancom) [inline src/helpers/fetch.js]: I don't really think I still have this path.

Comment (const-andreyLu) [reply src/helpers/fetch.js]: fixed

Comment (const-andreyLu) [reply src/helpers/fetch.js]: fixed

**Pull request merged**